Classes/Repository/RepositorySummaryCell.swift

```diff
@@ -0,0 +1,86 @@
+//
+
+import UIKit
+import SnapKit
+
+final class RepositorySummaryCell: UICollectionViewCell {
+    
+    static let labelInset = UIEdgeInsets(
+        top: Styles.Sizes.rowSpacing,
+        left: Styles.Sizes.icon.width + 2*Styles.Sizes.columnSpacing,
+        bottom: Styles.Fonts.secondary.lineHeight + 2*Styles.Sizes.rowSpacing,
+        right: Styles.Sizes.gutter
+    )
+    
+    private let reasonImageView = UIImageView()
+    private let titleLabel = UILabel()
+    private let secondaryLabel = UILabel()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        accessibilityTraits |= UIAccessibilityTraitButton
+        isAccessibilityElement = true
+        
+        contentView.backgroundColor = .white
+        
+        titleLabel.numberOfLines = 0
+        contentView.addSubview(titleLabel)
+        titleLabel.snp.makeConstraints { make in
+            make.top.equalTo(Styles.Sizes.rowSpacing)
+            make.left.equalTo(RepositorySummaryCell.labelInset.left)
+            make.right.equalTo(-RepositorySummaryCell.labelInset.right)
+        }
+        
+        reasonImageView.backgroundColor = .clear
+        reasonImageView.contentMode = .scaleAspectFit
+        reasonImageView.tintColor = Styles.Colors.Blue.medium.color
+        contentView.addSubview(reasonImageView)
+        reasonImageView.snp.makeConstraints { make in
+            make.size.equalTo(Styles.Sizes.icon)
+            make.centerY.equalToSuperview()
+            make.left.equalTo(Styles.Sizes.columnSpacing)
+        }
+        
+        secondaryLabel.numberOfLines = 1
+        secondaryLabel.font = Styles.Fonts.secondary
+        secondaryLabel.textColor = Styles.Colors.Gray.light.color
+        contentView.addSubview(secondaryLabel)
+        secondaryLabel.snp.makeConstraints { make in
+            make.top.equalTo(titleLabel.snp.bottom).offset(Styles.Sizes.rowSpacing)
+            make.left.equalTo(RepositorySummaryCell.labelInset.left)
+            make.right.equalTo(-RepositorySummaryCell.labelInset.right)
+        }
+                
+        addBorder(.bottom, left: RepositorySummaryCell.labelInset.left)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func configure(result: IssueSummaryType) {
+        titleLabel.attributedText = result.attributedTitle.attributedText
+        
+        let format = NSLocalizedString("#%d opened %@ by %@", comment: "")
+        secondaryLabel.text = String.localizedStringWithFormat(format, result.number, result.createdAtDate?.agoString ?? "", result.authorName ?? Strings.unknown)
+        
+        reasonImageView.image = result.stateIcon
+        reasonImageView.tintColor = result.stateColor
+    }
+    
+    override var accessibilityLabel: String? {
+        get {
+            return contentView.subviews
+                .flatMap { $0.accessibilityLabel }
+                .reduce("", { $0 + ".\n" + $1 })
```

Comment:
And because Objc, you have this awkward `get{} set{}` with the empty `set{}`. 🤷‍♂️

Do maybe want to do a PR to change all these to use string interpolation (as opposed to the plussing) to prevent the compiler from complaining about the expression being too complex.